Allow catch/delegate-less trys (#157)
WebAssembly/exception-handling: proposals/exception-handling/Exceptions.md

A try-catch block contains zero or more `catch` blocks and zero or one
`catch_all` block. All `catch` blocks must precede the `catch_all` block, if
any. The `catch`/`catch_all` instructions (within the try construct) are called
the _catching_ instructions. There may not be any `catch` or `catch_all` blocks

> after a `try`, at which point the whole `try` block is effectively a regular

Reviewer: language nit:

```suggestion
after a `try`, in which case the whole `try` block is effectively a regular
```
